lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -233,7 +233,7 @@ sub read_test_modules {
 }
 
 sub show {
-    my ($self) = @_;
+    my ($self, $limit_previous) = @_;
```

Comment:
there are no arguments to mojo controllers. So if you want to define an undefined variable, just do so :)